src/pal/src/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ endif()
 
 # Include directories
 
-include_directories(include)
+include_directories(include ${CoreCLR_SOURCE_DIR}/src/inc)
```

Comment:
I don't feel good about exposing all the CLR header files to PAL. And another issue is that it makes PAL depend on CoreCLR. This reverse dependency is strange and it would also complicate usage of PAL outside of CoreCLR (there are such cases).
I think would rather duplicate the CORJIT_FLAGS in PAL and added a compile time verification that they match.

Reply:
My original design was to add another layer with a separate set of flags, because I thought this design would not be acceptable.  

The ugliness of having this defined in three places (VM, JIT, and PAL) is annoying.  

+ Maybe I could move coreclr/src/inc/corjitflags.h to `coreclr/src/inc/pal/corjitflags.h` ? Still a reverse dependency, but it eliminates the inclusion of the whole `coreclr/src/inc` directory.
+ Maybe I can make CMake copy corjitflags into a pal include directory in the build tree, then include that....

I added the HAS_CORJITFLAGS_H so that PAL could easily turn this off on other platforms.